apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -352,20 +358,34 @@ class ReviewTab extends Component {
   }
 
   renderCommentEditor(forceRecreateEditorKey) {
-    if (this.state.isReadyForReview || this.props.viewAs === ViewType.Teacher) {
+    if (
+      !this.state.authorizationError &&
+      (this.state.isReadyForReview || this.props.viewAs === ViewType.Teacher)
+    ) {
       return (
         <CommentEditor
           onNewCommentSubmit={this.onNewCommentSubmit}
           key={forceRecreateEditorKey}
         />
       );
     }
-
-    return (
-      <div style={styles.messageText}>
-        {javalabMsg.disabledPeerReviewMessage()}
-      </div>
-    );
+    if (this.state.authorizationError) {
+      // this error messages is displayed if a student is trying to write a comment
+      // on a peer's project and the project has peer review disabled.
+      return (
+        <div style={{...styles.reviewDisabledText, ...styles.messageText}}>
+          {javalabMsg.peerReviewDisabled()}
+        </div>
+      );
+    } else {
+      // this message is displayed if a student is viewing their own project that has
+      // peer review disabled
+      return (
+        <div style={styles.messageText}>
+          {javalabMsg.disabledPeerReviewMessage()}
```

Comment:
nit: might be worth changing one of these string key names since they sound somewhat similar...maybe `peerReviewDisabledError`? or `disabledPeerReviewMessage` -> `enablePeerReviewMessage` (which then does seem similar to `enablePeerReview`)? not a huge deal since it's well documented here in the code, but could be helpful.

Reply:
the reason I didn't change the old one is I think if you change the key we would lose any existing translations. I can try to think of a more specific name for the new one

Reply:
renamed the new one to include 'error'